core/src/main/java/org/opensearch/sql/datasource/model/DataSourceType.java

```diff
@@ -8,8 +8,8 @@
 public enum DataSourceType {
   PROMETHEUS("prometheus"),
   OPENSEARCH("opensearch"),
-  JDBC("jdbc");
-
+  JDBC("jdbc"),
```

Comment:
not releated to this PR.  we do not use JDBC right? remove it.

Reply:
done